### PR TITLE
Implement new WorldNameChecker

### PR DESCRIPTION
**Pull request by benwoo1110**
Allow better preliminary checking of world name and folder status. Mainly targeted to ACF and improving command output, but bring it forward to split the journey to ACF a more manageable one.

### CODE REVIEW

**Review by dumptruckman — APPROVED** [review]
Very nice.

File: src/main/java/com/onarandombox/MultiverseCore/utils/WorldManager.java [hunk 1]
```diff
@@ -130,7 +130,7 @@ public boolean cloneWorld(String oldName, String newName) {
         }
 
         // Check for valid world name
-        if (!(isValidWorldName(oldName) && isValidWorldName(newName))) {
+        if (!(WorldNameChecker.isValidWorldName(oldName) && WorldNameChecker.isValidWorldName(oldName))) {
```
Comment (nicegamer7) [inline src/main/java/com/onarandombox/MultiverseCore/utils/WorldManager.java:133]: Shouldn't this be:
```diff
-if (!(WorldNameChecker.isValidWorldName(oldName) && WorldNameChecker.isValidWorldName(oldName))) {
+if (!(WorldNameChecker.isValidWorldName(oldName) && WorldNameChecker.isValidWorldName(newName))) {
```

Comment (benwoo1110) [reply src/main/java/com/onarandombox/MultiverseCore/utils/WorldManager.java:133]: Probably a copy paste errors, can open a PR to fix it

**Review by nicegamer7 — COMMENTED** [review]
Looks good 👍.

**Pull request merged**